src/domain/customers/CustomerDetailsComponent.tsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+
+import Grid from '../../common/grid/Grid';
+import styles from './customerList.module.scss';
+import Paragraph from '../../common/paragraph/Paragraph';
+
+export interface CustomerData {
+  comment?: string;
+  contactMethod?: string;
+  email?: string;
+  firstName?: string;
+  id: string;
+  image?: string;
+  invoicingType?: string;
+  language?: string;
+  lastName?: string;
+  nickname?: string;
+  phone?: string;
+}
+
+const CustomerDetails: React.SFC<CustomerData | any> = ({ data }: any) => {
```

Comment:
Why do you need `any`? It's better to avoid it as possible.

Reply:
I dont have access to type definitions yet. Any types shall be removed in another PR.

Reply:
Can you leave a `TODO` comment for fixing it?